src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2598,6 +2598,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             // closed. We can just do it whenever.
             _AsyncCloseConnection();
 
+            {
+                // We lock the terminal here to make sure it isn't still being

```

Comment:
```suggestion
                // GH#8734:
                // We lock the terminal here to make sure it isn't still being
```

Only suggestion is to add a note to the original issue, because I'm _certain_ we're not going to remember where this came from